.github/workflows/_run-pact-provider-tests.yml

```diff
@@ -27,6 +27,9 @@ jobs:
     runs-on: ubuntu-18.04
 
     steps:
+      - name: Get Provider SHA
```

Comment:
If you do this step after the checkout step the local filesystem will be pay-adminusers so you should be able to just do `git rev-parse HEAD`

Reply:
Changed. Let's give it a go.